manifests/c/calendulish/SteamToolsNG/latest/calendulish.SteamToolsNG.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: calendulish.SteamToolsNG
+PackageVersion: latest

```

Latest is not a valid version. The version should match what is written into control panel
```suggestion
PackageVersion: 3.0
```
After this change, the version folder will need to be renamed